src/containers/resources/ResourcesPage.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * @flow
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import { Colors, Typography } from 'lattice-ui-kit';
+
+import { RESOURCES, SUBTITLES } from '../../utils/constants/ResourcesConstants';
+import { trackLinkClick } from '../../utils/AnalyticsUtils';
+import { STATE } from '../../utils/constants/StateConstants';
+import { LABELS } from '../../utils/constants/Labels';
+import { getRenderTextFn } from '../../utils/AppUtils';
+import { ContentOuterWrapper, ContentWrapper, TextLink } from '../../components/layout';
+
+const { NEUTRAL } = Colors;
+
+const {
+  CALIFORNIA_RESOURCE_AND_REFERRAL_NETWORK,
+  CALIFORNIA_COVID19_CHILDCARE_RESPONSE,
+  CALIFORNIA_PARENT_AND_FAMILY_RESOURCES,
+  FAMILY_CHILD_CARE_HOME_LICENSING_INFORMATION,
+} = SUBTITLES;
+
+const Wrapper = styled(ContentWrapper)`
+  padding: 30px !important;
+  background-color: white;
+
+  h6 {
+    margin: 20px 0 5px;
+  }
+`;
+
+const Description = styled(Typography)`
+  color: ${NEUTRAL.N500};
+`;
+
+type Props = {
+  renderText :(currentLanguage :string) => string;
```

Comment:
Type annotation is incorrect. The function does not take a string. It takes a language translation object. I recommend creating a global language type:

```
// src/types.js

type Translation = {
  en :string;
  es :string;
};

export type {
  Translation,
};

// ResourcesPage.js

import type { Translation } from '../../types';

type Props = {
  getText :(translation :Translation) => string;
};
```